pkg/hubble/relay/server/server_test.go

```diff
@@ -43,12 +43,7 @@ import (
 	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
 )
 
-var log *logrus.Logger
-
-func init() {
-	log = logrus.New()
-	log.SetOutput(io.Discard)
-}
+var log = slog.New(slog.NewTextHandler(io.Discard, nil))
```

Comment:
no `hivetest.Logger` here? There are not that many references to the global `log` in the file.

Reply:
Same as before, I just tried to keep the same behavior as before, that's why it's still a io.Discard.